lessons/2_more_strings_and_functions/lesson.py

```diff
@@ -110,21 +110,39 @@ def my_cool_function(pet_name):
 # A. Use the function named multiply_and_subtract_five. Given that num1 is equal to 7 and num2 is 3,
 #    what do you expect the output of the function to be? Call the function and test it.
 # # ANSWER A GOES HERE
+# # # MH guess: (7*3)-5=16
 # # ANSWER A CODE GOES HERE
+print("the answer to exercise 1A is:",multiply_and_subtract_five(7,3))
```

Comment:
This works as it is but I want to make sure that you are aware of other ways to format strings for print statements in Python.

There is the format function:

name = "Emily"
food = "tacos"
print("{} would like to eat some {}".format(name, food))
where the {} will be formatted with the data in the order found.

There is also my personal favorite, f-string:

print("{name} would like to eat some {food}.")

Reply:
should i reply here? or should i leave my comments in the code?

in this case i am doing both....... but let me know what is *better*

anyway, i can't get the f-string to work.

Reply:
Reply wherever!

I think there is a preference for f-strings and .format() in the Python world but I'm not sure it matters?